### PR TITLE
Coil-Base-1.3.2

### PR DESCRIPTION
**Pull request by 4brunu**

### CODE REVIEW

File: Android/CoilBase/build.cake [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+
+var AAR_VERSION = "1.3.2";
+var AAR_URL = $"https://repo1.maven.org/maven2/io/coil-kt/coil-base/{AAR_VERSION}/coil-base-{AAR_VERSION}.aar";
+
+Task ("externals")
+	.Does (() =>
+{
+	EnsureDirectoryExists ("./externals");
+	
+	DownloadFile(AAR_URL, $"./externals/coil-base-{AAR_VERSION}.jar");
```
Comment (moljac) [inline Android/CoilBase/build.cake]: it is not jar, but aar. Thus nothing was surfaced....

i fixed it already. Working on metadata for namespaces etc...

**Review by moljac — COMMENTED** [review]
several fixes were needed.

File: Android/CoilBase/build.cake (hunk 1 above)
Comment (4brunu) [reply Android/CoilBase/build.cake]: I miss those, thanks for fixing those.
Based on the fixes you made, I will check the other open PRs

Comment (4brunu) [reply Android/CoilBase/build.cake]: Done, updated all open PRs.
https://github.com/xamarin/XamarinComponents/pull/1251
https://github.com/xamarin/XamarinComponents/pull/1246
https://github.com/xamarin/XamarinComponents/pull/1244
https://github.com/xamarin/XamarinComponents/pull/1245

**Comment by 4brunu** [comment]
@moljac is any more changes required to merge and publish this package? Thanks

**Review by moljac — APPROVED** [review]
LGTM

File: Android/CoilBase/cgmanifest.json [hunk 2]
```diff
@@ -0,0 +1,16 @@
+{
+  "Registrations": [
+    {
+      "Component": {
+        "Type": "Maven",
+        "Maven": {
+          "ArtifactId": "coil-base",
+          "GroupId": "io.coil-kt",
+          "Version": "1.3.2",
+          "NuGetId": "Xamarin.Io.Coil.Kt.Coil.Base"
```
Comment (4brunu) [inline Android/CoilBase/cgmanifest.json:10]: @moljac I think this field should also be updated to `Xamarin.Io.CoilKt.CoilBase`, or not?

Comment (moljac) [reply Android/CoilBase/cgmanifest.json:10]: yeah. thanks!

not critical for publishing nuget. will do it on main branch directly.

Comment (4brunu) [reply Android/CoilBase/cgmanifest.json:10]: Nice.
To make it easier for you I updated https://github.com/xamarin/XamarinComponents/pull/1251 with all the fixes you made in this branch .

**Pull request merged**